.travis.yml

```diff
@@ -11,6 +11,8 @@ env:
     - DATABASE_URL='postgres://postgres@localhost/groups'
   matrix:
     - DJANGO='django~=1.8.0'
+    - DJANGO='django~=1.9.0'
```

Comment:
1.9 is no longer receiving security updates, so it might not be wise to add this. Also, django-polymorphic 1.2 adds support for django 1.11, so that might be worth considering.

Nitpicks aside, materially, this LGTM `:)`